dimagi/data/commcare/pricing/support.py

```diff
@@ -63,17 +63,50 @@
         ),
         support=Support(
             standard=ugettext_lazy(
-                "3 Business Days"
+                ""
```

Comment:
actually, why are these being left blank instead of just being removed?

Reply:
Here we should to provide 4 property values either true/false or text value. So we keep as blank(blank string) for meets the criteria.